### ISSUE TITLE
file structure suggestion

### ISSUE THREAD

**Issue by jonahkagan** [scotch/angular-brunch-seed#4]
Some suggestions for changes to the file structure of the seed:
1. Rename `app/app` to `app/scripts` just to avoid confusion. I think it should have as few dirs with the same name as possible, especially if the dirs are nested.
2. Remove the `app/assets/partials/app` dir and move partials into `app/assets/partials` (same reason as 1.).

These might have some side effects elsewhere in the seed where these two paths are used.

**Comment by kylefinley** [comment]
Here's what I was thinking, but i'm hoping that there's a better way.

I agree that having `app` multiple places is a bit confusing. Maybe it could be change to the application actual name "nameofproject" or whatever. The reason that I'm using  `app` in `app/app`, `partials/app` and `styles/app.less` is to allow for third parties.

For example if I have a third party app name `sapling` I would like to include it like so: 

```
app /
  assets /
    partials /
      app /
        nav.jade
      sapling /
         account /
            login.jade
            signup.jade
   app /
     controllers.coffee
     ....
   vendor /
      scripts /
         sapling /
```

Maybe it's ok to have the partials belonging to the app in the root 'partials' while third party apps will be placed in a folder. 
I do really like the idea of changing `app/app` to `app/scripts`. 

The more that I think about it the problem really comes down to the partials directory. There's really no easy way to drop in third party code that uses templates, styles. What do you think the best way to handle third-party apps would be?

**Comment by jonahkagan** [comment]
Hm, I'm wondering what the use case would really be for dropping in third-party assets? I've never really heard of third-party templates, just starters or skeletons.

**Comment by kylefinley** [comment]
Django for example has this setup. You can include third party apps the are structured like:

```
django-thirdparty /
   models.py
   views.py
   templates /
     something.html
```

Maybe it's better to wait on brunch support on this:
https://github.com/brunch/brunch/issues/281

I like your restructuring, so for the time being I'll make the changes

**Comment by kylefinley** [comment]
Fixed in ad9862006f510551974f000656cd11d80e49d705 Thanks @jonahkagan
